HDF5Examples/C/H5FLT/tfiles/h5ex_d_bitgroom.ddl

```diff
@@ -0,0 +1,287 @@
+HDF5 "h5ex_d_bitgroom.h5" {
+GROUP "/" {
+   DATASET "DS1" {
+      DATATYPE  H5T_IEEE_F32LE
+      DATASPACE  SIMPLE { ( 32, 64 ) / ( 32, 64 ) }
+      STORAGE_LAYOUT {
+         CHUNKED ( 4, 8 )
+         SIZE 8192 (1.000:1 COMPRESSION)
+      }
+      FILTERS {
+         USER_DEFINED_FILTER {
+            FILTER_ID 32022
+            COMMENT BitGroom filter (Zender, 2016 GMD: http://www.geosci-model-dev.net/9/3199/2016)
```

Comment:
Use https.

Reply:
That usage is from the original code  and could cause issues with current netcdf files that use that filter.